cobc/codegen.c

```diff
@@ -6170,17 +6275,25 @@ output_initialize (struct cb_initialize *p)
 	if (type == INITIALIZE_NONE) {
 		return;
 	}
+
+	/* TODO: if cb_default_byte >= 0 do a huge memset first, then only
+	         emit setting for fields that need it (VALUE clause or
+	         special category - in general: not matching cb_default_byte);
+	         similar for cb_default_byte == -2 (just without the
+	         initial huge memset) */
```

Comment:
as other added TODOs: please include in the "to be inspected after merge" list; this specific one does have quite the optimization potential (likely the last one open for explicit/implicit `INITIALIZE`, before going into threading)
`

Reply:
Done.